WALLET.md

```diff
@@ -438,12 +439,18 @@ If you're using an Android simulator and the device / app is not showing up,
 navigate to settings (gear icon in the bottom left) and ensure the Android SDK
 location points to the same location as the $ANDROID_HOME environment variable.
 
-### App Profiling
+### App Profiling with react-devtools
 
-Run `yarn run react-devtools`. It should automatically connect to the running app, and includes a profiler (second tab). Start recording with the profiler, use the app, and then stop recording.
+From Flipper select React DevTools Plugin then while the app is running locally, or run `yarn run react-devtools` in the wallet root folder. It should automatically connect to the running app and includes a profiler (second tab). Start recording with the profiler, use the app and then stop recording. If running from the terminal, Flipper cannot be run at the same time.
```

Comment:
```suggestion
From Flipper select React DevTools Plugin while the app is running locally, or run `yarn run react-devtools` in the wallet root folder. It should automatically connect to the running app and includes a profiler (second tab). Start recording with the profiler, use the app and then stop recording. If running from the terminal, Flipper cannot be run at the same time.
```

Reply:
Changed in 878e5be8efbc6a25d4311cbb883665e9c096b835!